puppet/zulip_ops/manifests/postgres_common.pp

```diff
@@ -6,20 +6,21 @@
                                  "pv",
                                  "python3-pip",
                                  "python-pip",
-                                 # "python3-gevent", # missing on trusty
-                                 "python-gevent",
                                  # Postgres Nagios check plugin
                                  "check-postgres",
                                  ]
   package { $internal_postgres_packages: ensure => "installed" }
 
+  exec {"pip3_ensure_latest":
+    command => "/usr/bin/pip3 install -U pip==9.0.1",
+    creates => "/usr/local/bin/pip3",
+    require => Package['python3-pip'],
+  }
+
   exec {"pip_wal-e":
-    # On trusty, there is no python3-boto or python3-gevent package,
-    # so we keep our `wal-e` explicitly on Python 2 for now.
-    command  => "/usr/bin/pip2 install git+git://github.com/zbenjamin/wal-e.git#egg=wal-e",
+    command  => "/usr/local/bin/pip3 install 'boto==2.0.0' 'gevent==1.2.2' git+git://github.com/zbenjamin/wal-e.git#egg=wal-e",
```

Comment:
Does that version of wal-e actually run on Python 3?  

We might need to upgrade the version as well.

Reply:
Tried a Travis run on that version, failed because tox expected 2.7. So I decided to edit the commit to use the package vendored by wal-e, released at dec 2016.

Reply:
OK.  Note that both the puppet code and our wal-e configuration has no automated tests whatsoever.